circle.yml

```diff
@@ -0,0 +1,61 @@
+machine:
+  environment:
+    DOCKER_IMAGE_NAME: prom/memcached_exporter
```

Comment:
Looks like docker image with `_` works, so I don't understand why we have `node-exporer` and `mysqld-exporter` ...